src/popup/components/diagnostic-view-toggle.tsx

```diff
@@ -184,6 +184,10 @@ export class DiagnosticViewToggle extends React.Component<
     }
 
     private getCommandShortcut(): string {
+        if (!this.configuration.chromeCommand) {
+            return '';
```

Comment:
I think you can return null here instead. Generally for strings, we try not to use empty strings because they _should_ be valid values rather than cases like this where no such command shortcut exists.